### PR TITLE
[31293] Resolve Web Enable Install errors: Correct defined Web Port

### PR DESCRIPTION
**Pull request by andersdd**
if that port has already been used

### CODE REVIEW

File: functions/setup.fun [hunk 1]
```diff
@@ -540,6 +540,11 @@ config_webclient_scripts() {
 get_environment() {
   webenable_database
 
+  # Check if WEBAPI port is in use and increment if necessary
```
Comment (davidbeauchamp) [inline functions/setup.fun]: This is a function in common.sh to check if its open as well https://github.com/xtuple/xtuple-admin-utility/blob/master/common.sh#L209-L211

File: functions/setup.fun [hunk 2]
```diff
@@ -540,6 +540,11 @@ config_webclient_scripts() {
...
+  while [ -n "`lsof -Pi :${WEBAPI_PORT} -sTCP:LISTEN -t`" ]; do
+    let WEBAPI_PORT=WEBAPI_PORT+1
+  done
```
Comment (gilmoskowitz) [inline functions/setup.fun]:
```suggestion
  while ! is_port_open TCP $WEBAPI_PORT ; do
    (( WEBAPI_PORT++ ))
```

File: functions/setup.fun (hunk 1 above)
Comment (andersdd) [reply functions/setup.fun]: @davidbeauchamp I'm seeming to have difficulties with the `is_port_open` function in `common.sh`.  It seems to clock up to very, very high port numbers and I don't think all the lower numbers are open.  The  method of echoing /dev/tcp/localhost/port# does not work on an Ubuntu 18.04 server

**Comment by gilmoskowitz** [comment]
@andersdd https://github.com/andersdd/xtuple-utility/pull/4 should resolve the issues with `is_port_open`

**Pull request merged**